tests/core/pyspec/eth2spec/test/phase0/rewards/test_random.py

```diff
@@ -39,7 +39,15 @@ def test_full_random_3(spec, state):
 @with_custom_state(balances_fn=low_balances, threshold_fn=lambda spec: spec.EJECTION_BALANCE)
 @spec_test
 @single_phase
-def test_full_random_low_balances(spec, state):
+def test_full_random_low_balances_0(spec, state):
+    yield from rewards_helpers.run_test_full_random(spec, state)
+
+
+@with_all_phases
+@with_custom_state(balances_fn=low_balances, threshold_fn=lambda spec: 0)
```

Comment:
what's the intention here?

it looks like `threshold_fn` only affects who is active at this test's genesis -- so `test..._0` and `test..._1` as far as i can tell provide the same `state` here....

Reply:
ah, you're right. I thought it was 1.8ETH as "low balance" but it's 18 ETH so the threshold fn can be the default